src/video/sdl2_v.cpp

```diff
@@ -507,9 +483,7 @@ bool VideoDriver_SDL_Base::PollEvent()
 				// mouse entered the window, enable cursor
 				_cursor.in_window = true;
 #ifdef __EMSCRIPTEN__
-				/* Disable relative mouse mode for the first mouse motion,
-				 * so we can pick up the absolutely position again. */
-				_cursor_new_in_window = true;
+				// ensure pointer lock will not occur
```

Comment:
please use `/* */` style comments, and start with a capital and end with a dot. I know, nitpicking, sorry :)